src/client/mesh_generator_thread.cpp

```diff
@@ -207,8 +204,7 @@ CachedMapBlockData* MeshUpdateQueue::getCachedBlock(const v3s16 &p)
 
 void MeshUpdateQueue::fillDataFromMapBlockCache(QueuedMeshUpdate *q)
 {
-	MeshMakeData *data = new MeshMakeData(m_client, m_cache_enable_shaders,
-			m_cache_use_tangent_vertices);
+	MeshMakeData *data = new MeshMakeData(m_client, m_cache_enable_shaders);
```

Comment:
Perhaps the 3rd argument should be removed from `MeshMakeData()` https://github.com/minetest/minetest/search?q=MeshMakeData
... plus the related changes, such as removing `m_use_tangent_vertices` https://github.com/minetest/minetest/search?q=m_use_tangent_vertices
and `m_cache_use_tangent_vertices` https://github.com/minetest/minetest/search?q=m_cache_use_tangent_vertices

Reply:
I was debating that internally. Then decided not to, but you're right... I'll remove that too.

Reply:
Done. Removed even more code.